ui/lib/kv/addon/components/page/list.hbs

```diff
@@ -58,7 +58,7 @@
               @color="secondary"
               type="submit"
               disabled={{not this.secretPath}}
-              data-test-get-secret-detail
+              data-test-submit-button
```

Comment:
I think our general selector is `data-test-save` FWIW

Reply:
I was initially trying to differentiate from a form save button because I couldn't remember where this was rendered. I can update!